api_app/api/routes/airlock_resource_helpers.py

```diff
@@ -96,3 +98,36 @@ def check_email_exists(role_assignment_details: defaultdict(list)):
     if "owner_emails" not in role_assignment_details or not role_assignment_details["owner_emails"]:
         logging.error('Creating an airlock request but the workspace owner does not have an email address.')
         raise HTTPException(status_code=status.HTTP_417_EXPECTATION_FAILED, detail=strings.AIRLOCK_NO_OWNER_EMAIL)
+
+
+def get_airlock_requests_by_user_and_workspace(user: User, workspace: Workspace, airlock_request_repo: AirlockRequestRepository,
+                                               initiator: str = None, type: AirlockRequestType = None, status: AirlockRequestStatus = None, awaiting_my_review: bool = None) -> List[AirlockRequest]:
+    if awaiting_my_review:
```

Comment:
this variable is used only as a query parameter, we have to look at it from the user's perspective,
`GET ...requests?awaiting_my_review=true`
`GET ...requests?awaiting_current_user_review=true`
I think `awaiting_current_user_review` is a little confusing, I chose `awaiting_my_review` because it's the name of the checkbox in the UI

Reply:
let's simply call it ```awaiting_for_review``` and the function name ```get_airlock_requests_by_user_and_workspace``` better be ```get_airlock_requests``` as there are many more parameters now.

Reply:
discussed on it, changed to 
`awaiting_current_user_review`